src/ResourceManager/Network/Commands.Network/ChangeLog.md

```diff
@@ -64,6 +64,18 @@
 ## Version 6.3.1
 * Enable peering across Virtual Networks in multiple Tenants for Set/Add-AzureRmVirtualNetworkPeering
 
+## Version 6.4.0-preview
```

Comment:
can you remove the version header here and move these cmdlets under the [Current Release](https://github.com/Azure/azure-powershell/blob/2dca9c00f64f993da1bf296614259e06359a77aa/src/ResourceManager/Network/Commands.Network/ChangeLog.md#current-release) section